portfolio/src/main/webapp/script.js

```diff
@@ -66,3 +66,22 @@ function createListElement(text) {
   liElement.innerText = text;
   return liElement;
 }
+
+function createTaskElement() {
```

Comment:
IIRC - these are comments not tasks right? Consider cleaning up this and the docstring above deleteComment

Reply:
I tried to change it, but it kept giving reference error in the console, and it appeared to not change over there. I am still trying to figure it out. Just letting you know

Reply:
Figured it out. :)